backend/mutations.py

```diff
@@ -329,25 +328,22 @@ def mutate(self, info, artwork_data=None):
             title = artwork_data.title,
             artist = artwork_data.artist,
             description = artwork_data.description,
-            pictures = artwork_data.pictures if artwork_data.pictures else [],
-            found_by = decodeId(artwork_data.found_by),
+            pictures = [artwork_data.picture_to_add] if artwork_data.picture_to_add else [],
+            found_by = decodeId(artwork_data.found_by),   # might wanna error check
```

Comment:
I think this might break the submission on the frontend. Is there a reason why you changed the schema here?